src/geo/ui/infowindow.js

```diff
@@ -119,12 +118,15 @@ var Infowindow = View.extend({
         this.$('.js-content').css('max-height', this.model.get('maxHeight') + 'px');
       }
 
-      // If the template is 'cover-enabled', load the cover
+      this._renderLoader();
+      this._startLoader();
+
       this._loadCover();
 
       if (!this.isLoadingData()) {
         this.model.trigger('domready', this, this.$el);
         this.trigger('domready', this, this.$el);
+        this._stopLoader();
```

Comment:
Why is it necessary to render/start loader in the previous lines if it's just going to be stopped a few lines later?